### PR TITLE
:clap: Make 'default install location' clearer in preferences UI

### PR DESCRIPTION
**Pull request by ESWAT**
Fixeds #941.

Adds a (default location) string next to the default location. __Needs translations__

screen shot 2016-11-02 at 22 10 15

### CODE REVIEW

**Comment by fasterthanlime** [comment]
Nice PR, thanks!

My only gripe is that the `(default location)` string should be localized :)

See how `t('preferences.category.label')` is used elsewhere in the component. You can find the corresponding strings in `en.json` — if you'd rather have me fix the PR, tell me and I'll merge as-is then clean up right after.

**Comment by fasterthanlime** [comment]
👍 Looks good to me, will merge soon!

**Comment by ESWAT** [comment]
I updated the branch to use `preferences.install_location.is_default_short` since that already exists.

Wasn’t sure what to do with the first commit since it would have required new translations, or combining `preferences.install_location.is_default_short` + `preferences.install_location.path` to result in `Default Location`.

Either way if you want to merge and take over, feel free. :)

**Comment by fasterthanlime** [comment]
> Wasn’t sure what to do with the first commit

One alternative is to change the string in `en.json` to `default location` — this will flag the string as "needs review" for translators, which will then able to adjust it accordingly for their language :)

**Comment by ESWAT** [comment]
Ah, will keep that in mind for the future. :)

**Comment by ESWAT** [comment]
Made one last change to the default location label. Since the word `location` is used fairly often in this section I think it’s fine to just leave `(default)` as the label.

tl;dr ready for merge

screen shot 2016-11-04 at 09 54 34

**Comment by fasterthanlime** [comment]
Note that it's taking longer for me to merge this than it would usually, since I'm working on porting the whole application to typescript (see #951) — but don't worry, your work is not lost :) Just apologizing for the delay.

**Comment by fasterthanlime** [comment]
Merged manually!
